### ISSUE TITLE
bicon freezes

### ISSUE THREAD

**Issue by abakh** [behdad/bicon#23]
with bicon on fbterm on a knoppix tty
`cat /dev/random | tee /dev/fb0`

causes bicon.bin to use 100% cpu.

**Comment by behdad** [comment]
Yep.  Don't do that. :)

BiCon is a hack.  Use it for what it works.  Don't use it for anything else.

**Comment by abakh** [comment]
yes, but it is not expecting something from bicon that bicon doesn't.
it is abnormal that a program which handles text input/output gets freezed by non-related framebuffer stuff
